src/calchartdoc.cpp

```diff
@@ -279,6 +281,25 @@ wxInputStream& CalChartDoc::LoadObject(wxInputStream& stream)
 }
 #endif
 
+bool CalChartDoc::exportViewerFile(const wxString& filepath) {    
+    JSONElement mainObjectElement = JSONElement::makeObject();
+    
+    JSONDataObjectAccessor mainObject = mainObjectElement;
```

Comment:
`JSONDataObjectAccessor` has a constructor of the form: `JSONDataObjectAccessor(const JSONElement& element)`. Because this constructor isn't marked as `explicit`, it allows us to implicitly convert from a `JSONElement` to a `JSONDataObjectAccessor`, which is what is happening here. If you wrote `JSONDataObjectAccessor mainObject(mainObjectElement)`, it would do the same thing.